### ISSUE TITLE
Doesn't parse @@

### ISSUE THREAD

**Issue by tbm** [glasserc/ledger-to-beancount#6]
ledger allows you to specify the total with @@ (rather than the unit price with @). The script
doesn't like this:

```
2018-01-01 * Opening balance: property
    ; :opening-balance:
    Assets:Property          2 FOO {{ 20.00 EUR}} @@ 20.00 EUR
    Equity:Opening balance
```

```
  File "/home/tbm/.local/lib/python3.5/site-packages/ledger_to_beancount/__init__.py", line 160, in translate_file
    (amount, price) = rest.split('@')
```

**Comment by glasserc** [comment]
I'm not really sure how to handle this. For sales, of course, it's easy to use total price instead of individual price. However, for purchases, I'd want to turn this into a cost basis, and doing so means doing math, which means a semantic, rather than syntactic, change. I've tried to avoid doing anything like that so far in the project.

**Comment by tbm** [comment]
Right, in ledger2beancount I ended up with an interpretation (aided by a config file that users can edit).

But if you don't want to go the semantic route, you should just transform it into a total price in beancount all the time.
